webhooks.md

```diff
@@ -0,0 +1,192 @@
+# Webhooks
+
```

Comment:
Missing some description like e.g.

Currently supported webhooks for operation on integrations. Action is triggered everytime when hotel updates the integration.

Reply:
The list should contain heading for each action and it should be a link. To be able to click on it.

* Integration created
* Integtration deleted